dev/NavigationView/Strings/en-us/Resources.resw

```diff
@@ -153,8 +153,8 @@
     <value>More Button</value>
     <comment>Automation name for the nav view more button when panel is on top</comment>
   </data>
-  <data name="NavigationOverflowButtonText" xml:space="preserve">
```

Comment:
I guess if we remove the `NavigationOverflowButtonText` entry from the en-US resource file, we should also do so for all the other language resource files for the NavigationView.

Reply:
@Felix-Dev If you remove it from here, the localization workflow will take care of removing the rest. We don't need to manually edit the other ones.

Reply:
@ranjeshj Very nice 😁

Reply:
Will removing this entry break people who have taken a copy of the previous template?

Reply:
@StephenLPeters What could be an issue here is that now we don't set the previous "More" text to the overflow button any longer. So those developers who created a copy of the `NavigationViewOverflowButtonStyleWhenPaneOnTop` overflow style now won't see the "More" text any longer (if they want that text, they need to manually add it to their style copy).

I could revert the commit f8da599 which removed setting the overflow button's text content bit then this code should be marked as such: Its only purpose would be to guarantee backwards compatibility and should be removed in the future in a major WinUI release.

Reply:
@ranjeshj and @YuliKl personally I think that if we can preserve backwards compatibility with the previous template we should, and we can break this preservation with winui 4.

Reply:
Ok, I reverted commit f8da599 just now.